runtime/lua/util.lua

```diff
@@ -0,0 +1,66 @@
+
+local util = {}
+
+vim = vim or {}
+
+util.echom = function(message)
+  vim.api.nvim_command('echom "' .. tostring(message) .. '"')
```

Comment:
Just remove this function and use `print()`. And you have escaping issues here in any case.